src/config/active-response.c

```diff
@@ -273,14 +383,23 @@ int ReadActiveResponses(XML_NODE node, void *d1, void *d2)
     /* Set a unique active response name */
     tmp_ar->name = (char *) calloc(OS_FLSIZE + 1, sizeof(char));
     if (!tmp_ar->name) {
-        merror_exit(MEM_ERROR, errno, strerror(errno));
+        if (output == NULL) {
+            merror_exit(MEM_ERROR, errno, strerror(errno));
+        } else {
+            snprintf(message, OS_FLSIZE,
+                "Could not acquire memory due to [(%d)-(%s)].",
+                errno, strerror(errno));
+            wm_strcat(output, message, '\n');
```

Comment:
The original code exited on this error, the new code keeps running, is this the intended behaviour?